app/controllers/CyaCreateRecordController.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import cats.data
+import com.google.inject.Inject
+import controllers.actions.{DataRequiredAction, DataRetrievalAction, IdentifierAction}
+import logging.Logging
+import models.{GoodsRecord, ValidationError}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import uk.gov.hmrc.play.bootstrap.binders.RedirectUrl
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import viewmodels.govuk.summarylist._
+import views.html.CyaCreateRecordView
+
+class CyaCreateRecordController @Inject() (
```

Comment:
Is it possible to put each journey into a package e.g.
profile package would contain all profile related files and the cya page for that
Similarly here we could have a createRecord pakage  - I wonder if we do that whether we can rename these to CheckYourAnswers as it'll be under a different package - don't worry about it if it's too much effort

Reply:
i've been doing the moving packages thing and it is quite a huge job with lots of file changes, at the very least i don't think it wants to be part of this ticket. if we want to restructure i think we should do this as a separate ticket and check with the other devs about what structure would suit us - also i think that since it goes against the scaffold pattern we probably might not even want to do it

Reply:
Agreed to not do this and leave it